cpp/src/parquet/encoding.h

```diff
@@ -466,4 +467,8 @@ std::unique_ptr<typename EncodingTraits<DType>::Decoder> MakeTypedDecoder(
   return std::unique_ptr<OutType>(dynamic_cast<OutType*>(base.release()));
 }
 
+Encoding::type ChooseFallbackEncoding(Type::type data_type,
```

Comment:
```suggestion
PARQUET_EXPORT Encoding::type ChooseFallbackEncoding(Type::type data_type,
```
IIUC, this may fix the Windows CI.

Reply:
Added it. I seem to be getting some other, actual, test failures at the moment. Looking into them.